lib/ash_graphql.ex

```diff
@@ -263,11 +263,13 @@ defmodule AshGraphql do
     end)
   end
 
-  def add_context(ctx, apis) do
+  def add_context(ctx, apis, options \\ []) do
+    empty_dataloader = Dataloader.new(options ++ [get_policy: :tuples])
```

Comment:
I think we should do `Keyword.put(options, :get_policy, :tuples)` here.

Reply:
Done